apps/humanatlas.io/src/app/pages/publications-page/publications-page.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="banner">
+  <img src="assets/images-v2/publications.png" alt="Publications Banner" />
+  <span class="banner-text">Publications</span>
+</div>
+
+<div class="page-content">
+  <div class="navigation-panel small">
+    <div class="navigation">Panel goes here</div>
+  </div>
+  <div class="page-sections">
+    @for (year of years(); track year) {
+      <hra-page-section>
+        <header-content>
+          <hra-section-link level="2" [anchor]="year" underlined> {{ year }} </hra-section-link>
```

Comment:
Don't rely on year being a valid html id. Use a method to convert strings into ids. Something like the following should work: (please test this, I might have made mistakes)
```ts
toAnchor(value: string): string {
  return value.replace(/[^\w:._-]/g, '-').replace(/^[^a-z]/i, 'id-$&');
}